src/server/worldserver/CommandLine/CliRunnable.cpp

```diff
@@ -91,7 +93,9 @@ void utf8print(void* /*arg*/, char const* str)
     if (!Utf8toWStr(str, strlen(str), wtemp_buf, wtemp_len))
         return;
 
-    wprintf(L"%s", wtemp_buf);
+    char temp_buf[6000];
```

Comment:
does a wchar_t[6000] fit in a char[6000] ? is there any C++17 feature or so that might replace the platform-specific CharToOemBuffW call ?

Reply:
hell no it doesnt, sizeof(wchar_t) is 2 on windows, 4 on linux

Reply:
wchar is cancer, avoid at all cost.

Reply:
Also this is going to warn about stack size